examples/municipality/src/run_municipality.py

```diff
@@ -1,10 +1,10 @@
-from rtctools_heat_network.workflows import EndScenarioSizingStagedHIGHS, run_end_scenario_sizing
+from rtctools_heat_network.workflows import EndScenarioSizingStaged, run_end_scenario_sizing
 
 if __name__ == "__main__":
     import time
 
     start_time = time.time()
 
-    solution = run_end_scenario_sizing(EndScenarioSizingStagedHIGHS)
+    solution = run_end_scenario_sizing(EndScenarioSizingStaged)
```

Comment:
Was this change on purpose? Not using EndScenarioSizingStagedHIGHS now

Reply:
I will revert this, probably happened during testing